projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/judicial-user/write-judicial-user-field.component.html

```diff
@@ -1,43 +1,21 @@
-<div class="form-group" [formGroup]="judicialUserFormGroup">
-  <label
-    [for]="id()"
-  >
-    <span
-      *ngIf="caseField.label"
-      class="form-label"
-    >
+<div class="form-group" [formGroup]="formGroup">
+  <label [for]="id()">
+    <span *ngIf="caseField.label" class="form-label">
       {{caseField | ccdFieldLabel}}
     </span>
   </label>
-  <span
-    *ngIf="caseField.hint_text"
-    class="form-hint"
-  >
+  <span *ngIf="caseField.hint_text" class="form-hint">
     {{caseField.hint_text}}
   </span>
-  <input
-    class="form-control bottom-30"
-    [id]="id()"
-    [formControl]="idamIdFormControl"
-    [matAutocomplete]="autoComplete"
-  >
-  <mat-autocomplete
-    autoActiveFirstOption
-    #autoComplete="matAutocomplete"
-  >
-    <mat-option
-      *ngFor="let judicialUser of filteredJudicialUsers"
-      class="select-option"
-      [ngClass]="{'hide-autocomplete': !showAutocomplete}"
-      [value]="judicialUser.idamId"
-      (onSelectionChange)="onSelectionChange(judicialUser)"
-    >
-      {{ judicialUser.fullName }} ({{ judicialUser.emailId }})
+  <input class="form-control bottom-30" [id]="id()"
+    [formControl]="judicialUserControl" [matAutocomplete]="autoComplete">
+  <mat-autocomplete autoActiveFirstOption #autoComplete="matAutocomplete" [displayWith]="displayJudicialUser">
+    <mat-option *ngFor="let judicialUser of filteredJudicialUsers$ | async"
+      class="select-option" [ngClass]="{'hide-autocomplete': !showAutocomplete}"
+      [value]="judicialUser" (onSelectionChange)="onSelectionChange($event)">
+      {{ judicialUser.fullName }}
```

Comment:
@Daniel-Lam - we have to display in the format _judicialUser.fullName (judicialUser.Email)_

Reply:
Hi @johnbenjamin-hmcts - ah, thanks for that... I'll correct that now and re-release